README.md

```diff
@@ -4,9 +4,15 @@
 [![Go Report Card](https://goreportcard.com/badge/prometheus-operator/prometheus-operator "Go Report Card")](https://goreportcard.com/report/prometheus-operator/prometheus-operator)
 [![Slack](https://img.shields.io/badge/join%20slack-%23prometheus--operator-brightgreen.svg)](http://slack.k8s.io/)
 
-**Project status: *beta*** Not all planned features are completed. The API, spec, status and other user facing objects may change, but in a backward compatible way.
+*Note: Project was previously known as coreos/prometheus-operator.*
```

Comment:
Is that necessary? 🤔 It has been quite a few years since this changed, I believe that people are used to it now

Reply:
I don't think so, I left it as it was there but I'm happy to remove it 😅

Reply:
Removed in 9b5018cff